sorc/ncep_post.fd/INITPOST_MPAS.F

```diff
@@ -2,7 +2,7 @@
 !> @brief initpost() initializes post for run.
 !>
 !> @author Russ Treadon W/NP2 @date 1993-11-10
-
+!
```

Comment:
@jaymes-kenyon Please add a change log. You should be listed as the author who initialized this subroutine.